main.go

```diff
@@ -122,34 +138,53 @@ func main() {
 		os.Exit(0)
 	}
 
+	if !*statsdListenUDP && !*statsdListenTCP {
+		log.Fatalln("At least one of UDP/TCP listeners should be specified.")
+	}
+
 	if *addSuffix {
 		log.Warnln("Warning: Using -statsd.add-suffix is discouraged. We recommend explicitly naming metrics appropriately in the mapping configuration.")
 	}
+
 	log.Infoln("Starting StatsD -> Prometheus Exporter", version.Info())
 	log.Infoln("Build context", version.BuildContext())
-	log.Infoln("Accepting StatsD Traffic on", *statsdListenAddress)
+	log.Infof("Accepting StatsD Traffic on %s, UDP %v, TCP %v", *statsdListenAddress, *statsdListenUDP, *statsdListenTCP)
```

Comment:
That's not longer printing the address it's listening on, but `true`/`false`, but would be fixed with the comment above.